Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -133,11 +138,17 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
     });
 
     return (
-      <View style={[ styles.appbar, style ]} {...viewProps}>
+      <Animated.View style={[
+          styles.appbar,
+          { height: APPBAR_HEIGHT + this.props.statusBarHeight },
```

Comment:
@corbt I don't think the ability to set custom height of statusbar is very important, and in 99% cases, just having the hidden prop is fine, but I think it's nice to have the flexibility. In addition, hardcoding the statusbar height is not ideal for other platforms implementing RN like WP.